pom.xml

```diff
@@ -166,111 +133,14 @@
             <plugin>
                 <groupId>com.exasol</groupId>
                 <artifactId>project-keeper-maven-plugin</artifactId>
-                <version>1.3.4</version>
+                <version>2.4.6</version>
                 <executions>
                     <execution>
                         <goals>
                             <goal>verify</goal>
                         </goals>
                     </execution>
                 </executions>
-                <configuration>
-                    <modules>
-                        <module>integration_tests</module>
-                        <module>maven_central</module>
-                    </modules>
-                    <linkReplacements>
-                        <linkReplacement>
-                            LICENSE-exasol-jdbc.txt|https://docs.exasol.com/connect_exasol/drivers/jdbc.htm
```

Comment:
Do you need to add this to .project-keeper.yml?

Reply:
The link checker says: yes.

And here I hoped that PK would do that for me. :-)

Reply:
Fixed.